### ISSUE TITLE
Task Error (WN) 

### ISSUE THREAD

**Issue by brucellino** [AAROC/DevOps#251]
Task failed on Munge key distribution: 

```
TASK: [Copy Munge key to the worker nodes] ************************************ 
failed: [ce01-node005.c4.csir.co.za] => {"failed": true}
msg: could not find src=/etc/munge/munge.key
```
The task is : 

```yaml
hosts: worker-nodes
  name: Copy Munge key to the worker nodes
  tasks:
  - name: Copy Munge key to the worker nodes
    copy:
      src: /etc/munge/munge.key
      dest: /etc/munge/munge.key
      mode: 0600
      owner: root
      group: root
```

**Comment by brucellino** [comment]
Diagnosis - permission on the source file `/etc/munge/munge.key` means that the user executing the play (`ansible`) cannot actually access the file, so it thinks it's not there. See e.g. https://github.com/ansible/ansible/issues/2303. Need to fix this in the CREAM-CE play.

**Comment by brucellino** [comment]
Fixed by afeff5b6d56be31bff332934cf74e5a950616069
